### PR TITLE
Add /api/v3/slas/licenses route

### PR DESCRIPTION
**Pull request by ekapassa**

### CODE REVIEW

**Comment by ekapassa** [comment]
@jbonnet made an update to the permissions. Please check it.

File: tng-router/config/sp_routes.yml [hunk 1]
```diff
@@ -118,23 +118,32 @@ paths:
...
     permissions:
     - role: '-'
-      verbs: get
+      verbs: 'options,get'
   /api/v3/slas/agreements(/?|/*): 
     site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/agreements
     permissions:
     - role: '-'
-      verbs: get
+      verbs: 'options,post,get,delete'
```
Comment (jbonnet) [inline tng-router/config/sp_routes.yml]: are you sure you want any user to be able to do this?

File: tng-router/config/sp_routes.yml [hunk 2]
```diff
@@ -118,23 +118,32 @@ paths:
...
     site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/violations
     permissions:
     - role: '-'
-      verbs: get
+      verbs: 'option,get'
+  /api/v3/slas/licenses(/?|/*): 
+    site: http://tng-sla-mgmt:8080/tng-sla-mgmt/api/slas/v1/licenses
+    permissions:
+    - role: 'admin'
+      verbs: 'option,post,get,delete'
```
Comment (jbonnet) [inline tng-router/config/sp_routes.yml]: options?

File: tng-router/config/sp_routes.yml [hunk 3]
```diff
@@ -118,23 +118,32 @@ paths:
...
+    - role: 'customer'
+      verbs: 'option,post,get'
```
Comment (jbonnet) [inline tng-router/config/sp_routes.yml]: optionS?

File: tng-router/config/sp_routes.yml (hunk 2 above)
Comment (jbonnet) [inline tng-router/config/sp_routes.yml]: optionS?

Comment (ekapassa) [reply tng-router/config/sp_routes.yml]: updated

File: tng-router/config/sp_routes.yml (hunk 1 above)
Comment (ekapassa) [reply tng-router/config/sp_routes.yml]: I gave it a second thought and I think you are right. It shouldn't have any user access to this...
Please check my update and let me know if you agree and this makes sense now.

**Pull request merged**